packages/ddex/entrypoint/src/components/Success.tsx

```diff
@@ -44,6 +61,15 @@ export const Success = () => {
       >
         {messages.access}
       </Text>
+      <Flex gap='m' justifyContent='center'>
+        {user ? data?.filter(app => distributorAppKeys.includes(app.address)).map(app =>
+          <DistributorCard
+            key={app.address}
+            appKey={app.address}
+            initialData={{...app, userId: user.id }}
+          />
+        ) : null}
+      </Flex>
```

Comment:
More of a design concern but I feel like having "authorized apps" and "settings" both be linked to the same place is kind of silly

Reply:
done!